src/copy.jl

```diff
@@ -36,11 +36,11 @@ reference of the copied model can be obtained by indexing the map with the
 reference of the corresponding reference of the original model.
 """
 struct ReferenceMap
-    model::Model
+    model::GenericModel
```

Comment:
```suggestion
    model::GenericModel{T}
```
Should this be a concrete type?

Reply:
Good catch

Reply:
Fixed